### ISSUE TITLE
Add maxColumns and removeEmptyRows parameters to ExcelHelper.ToDataTable() 

### ISSUE THREAD

**Issue by Ninjanaut** [WeihanLi/WeihanLi.Npoi#122]
**Is your feature request related to a problem? Please describe.**

Many times `ExcelHelper.ToDataTable()` returns `DataTable` like

![image](https://user-images.githubusercontent.com/44192515/130317079-5faa73f5-3afd-432d-b9bc-a34b9c5fdfd7.png)

* The issue is probably caused by that cells has applied formatting, but in practice users might be loading thousands of unnecessary columns and rows which also has performance impact.

or it does not load `DataTable` at all, because of weird `System.IndexOutOfRangeException`, when cell count is for example 5, but row count is 4 (I do not know how to create such an excel, but I attach an example with that exception
[ExcelWithOutOfRangeException.xlsx](https://github.com/WeihanLi/WeihanLi.Npoi/files/7025233/ExcelWithOutOfRangeException.xlsx)

![image](https://user-images.githubusercontent.com/44192515/130317120-829f05d4-8a49-467d-8390-543cdca3f6da.png)

![image](https://user-images.githubusercontent.com/44192515/130317132-6bbca549-dce0-45b4-bb1b-84fc5c12988e.png)

**Describe the solution you'd like**
So I thought an elegant solution would be to extend method signature of `ExcelHelper.ToDataTable()` with maxColumns and removeEmptyRows like

```csharp
public static DataTable ToDataTable(byte[] excelBytes, ExcelFormat excelFormat, bool removeEmptyRows = false, int? maxColumns = null)
```
**Describe alternatives you've considered**
Removing columns and rows can be done later when `DataTable` is already loaded (except that case throwing `System.IndexOutOfRangeException`) but it requires unnecessary additional loop through `DataTable` and is so common operation that it would be great to have it in one method and not have to write it separately. 

**Additional context**
if there will be an interest in these features, I'll be happy to send a pull request.

**Comment by WeihanLi** [comment]
@Ninjanaut, great thanks for your reporting and your valuable thoughts, sounds great to me, and it would be great if you could make a PR to fix it

**Comment by WeihanLi** [comment]
Should be closed by #123 , #124 
A new version 1.20.0 should work now, please report if this issue still exists, thanks.
